app/models/miq_alert.rb

```diff
@@ -844,4 +845,23 @@ def self.import_from_yaml(fd)
       stat
     end
   end
+
+  def hawkular_keys
+    aux = super
```

Comment:
Aaaahhh!! It's an "auxiliary" variable. Umm... It's an old habit that I have from my days developing in standard C. But `aux` may not be widely used. May be I should rename it to `temp` or use a more descriptive name like `persisted_value`.

I will change it if this PR is not going to be dropped as a result of [this comment](https://github.com/ManageIQ/manageiq/pull/15306#issuecomment-306792229).